<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="LAPTOP-1SFV9PNA" failures="1" tests="4" name="Parabank Tests" time="63.044" errors="0" timestamp="2025-08-28T11:26:04 IST">
  <testcase classname="com.capstone.tests.LoginTests" name="loginTest" time="4.75"/>
  <testcase classname="com.capstone.tests.LoginTests" name="loginTest" time="19.89"/>
  <testcase classname="com.capstone.tests.TransferTests" name="transferFlow" time="23.39">
    <failure type="java.lang.AssertionError" message="Transfer should complete expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Transfer should complete expected [true] but found [false]
at com.capstone.tests.TransferTests.transferFlow(TransferTests.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- transferFlow -->
  <testcase classname="com.capstone.tests.TransactionTests" name="transactionHistory" time="6.578"/>
  <testcase classname="com.capstone.tests.BaseTest" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.capstone.tests.BaseTest" name="@BeforeSuite beforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
  <testcase classname="com.capstone.tests.BaseTest" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.capstone.tests.BaseTest" name="@BeforeSuite beforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
  <testcase classname="com.capstone.tests.BaseTest" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.capstone.tests.BaseTest" name="@BeforeSuite beforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
</testsuite> <!-- Parabank Tests -->
